vars/sectionDeployToAKS.groovy

```diff
@@ -70,9 +73,11 @@ def call(params) {
         withTeamSecrets(config, environment) {
           stageWithAgent("Smoke Test - AKS ${environment}", product) {
             testEnv(aksUrl) {
-              pcr.callAround("smoketest:${environment}") {
-                timeoutWithMsg(time: 10, unit: 'MINUTES', action: 'Smoke Test - AKS') {
-                  builder.smokeTest()
+              pcr.callAround("smoketest-aks:${environment}") {
```

Comment:
why do you need a -aks version of this?

Reply:
I'm not sure, I will change it, I didn't actually write any of this

Reply:
ah there was some closures in the Jenkins file     
```groovy
before('smoketest-aks:idam-preview') {
        env.PREVIEW_ENVIRONMENT_NAME = 'preview'
        env.NONPROD_ENVIRONMENT_NAME = 'preview'
        env.PUBLIC_URL = 'https://idam-web-public.service.core-compute-preview.internal'
        println """\
            Using PREVIEW_ENVIRONMENT_NAME: ${env.PREVIEW_ENVIRONMENT_NAME}
            Using NONPROD_ENVIRONMENT_NAME: ${env.NONPROD_ENVIRONMENT_NAME}
            Using PUBLIC_URL: ${env.PUBLIC_URL}""".stripIndent()
    }
```

Probably from when we where deploying to AKS and whatever the previous app container was called in different environments so we needed to differentiate.